x-pack/plugins/secops/public/components/page/network/network_top_n_flow_table/__snapshots__/select_type.test.tsx.snap

```diff
@@ -0,0 +1,11 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`NetworkTopNFlow Select direction rendering it renders the default Authentication table 1`] = `
```

Comment:
This test still has ` Authentication table 1`, should that be network top n flow table?